llvm/lib/Demangle/MicrosoftDemangle.cpp

```diff
@@ -53,6 +53,18 @@ static bool consumeFront(std::string_view &S, std::string_view C) {
   return true;
 }
 
+static bool consumeFront(std::string_view &S, std::string_view CA,
```

Comment:
nit: I'm guessing the `C` in these parameter names is really for "character", based on the initial overload on line 42. Maybe we could find better names for the string_views (how about Prefix?)